### PR TITLE
Optimize docker compose

### PR DESCRIPTION
**Pull request by Simon-Tl**
### optimize
- Optimize the image storage address for faucet, explorer
- Optimize the README file
- ~~Failed to process the blockscan index.~~

### CODE REVIEW

File: docker-deploy/.env.explorer [hunk 1]
```diff
@@ -0,0 +1,92 @@
+ETHEREUM_JSONRPC_VARIANT=geth
+ETHEREUM_JSONRPC_HTTP_URL=http://localhost:8000/
+ETHEREUM_JSONRPC_TRACE_URL=http://localhost:8000/
```
Comment (Flouse) [inline docker-deploy/.env.explorer:3]:
```suggestion
ETHEREUM_JSONRPC_HTTP_URL=http://host.docker.internal:8000/
ETHEREUM_JSONRPC_TRACE_URL=http://host.docker.internal:8000/
```

**Review by Flouse — CHANGES_REQUESTED** [review]

File: docker-deploy/.env.faucet [hunk 2]
```diff
@@ -0,0 +1,12 @@
+AXON_FAUCET_REQUIRED_CONFIRMATIONS=10
+AXON_FAUCET_CLAIM_VALUE=1000000000000000000
+
+AXON_FAUCET_RPC_URL=http://localhost:8000
```
Comment (Flouse) [inline docker-deploy/.env.faucet:4]: Please check this URL too.

Comment (Simon-Tl) [reply docker-deploy/.env.faucet]: OK，inspected

**Comment by jiangxianliang007** [comment]
Axon repo https://github.com/axonweb3/axon/blob/main/docker-compose.yml already has a docker-compose to start a single Axon node.

Do you need a docker-compose that can start 4 axon nodes?

File: docker-deploy/docker-compose.yml [hunk 3]
```diff
@@ -16,62 +16,82 @@ services:
...
-    restart: always
-    privileged: true
-    deploy:
-      resources:
-        limits:
-          cpus: '1'
-          memory: 2048M
+
+  faucet-server:
+    image: ghcr.io/simon-tl/axon-faucet:latest
```
Comment (jiangxianliang007) [inline docker-deploy/docker-compose.yml:26]: image tag it's  recommended to specify the version, not the latest.

Comment (Simon-Tl) [reply docker-deploy/docker-compose.yml:26]: ok, this is just me testing using

**Comment by Flouse** [comment]
- replaced by https://github.com/axonweb3/axon-devops/pull/168

**Pull request closed without merging**